src/ClassSelectionMenu/index.jsx

```diff
@@ -54,9 +54,33 @@ const Number = styled("div")(({theme}) => ({
 
 export const ClassSelectionMenu = ({
   selectedCls,
+  preselectCls,
   regionClsList,
   onSelectCls,
 }) => {
+  useEffect(() => {
```

Comment:
I have now found out why this useEffect() call had been removed in the meantime (and we readded it earlier during the merge).
This is part of the problematic keyboard shortcuts, that Anett removed. So you have to remove all key related parts and check if the implementation still works.